Content.Server/SimpleStation14/EndOfRoundStats/EmitSound/EmitSoundStatSystem.cs

```diff
@@ -0,0 +1,99 @@
+using System.Diagnostics.CodeAnalysis;
+using Content.Server.GameTicking;
+using Content.Shared.GameTicking;
+using Content.Shared.SimpleStation14.EndOfRoundStats.EmitSound;
+using Content.Shared.Tag;
+
+namespace Content.Server.SimpleStation14.EndOfRoundStats.EmitSound;
+
+public sealed class EmitSoundStatSystem : EntitySystem
+{
+    Dictionary<SoundSources, int> soundsEmitted = new();
+
+    // This Enum must match the exact tag you're searching for.
+    // Adding a new tag to this Enum, and ensuring the localisation is set will automatically add it to the end of round stats.
+
+    private enum SoundSources
+    {
+        BikeHorn,
+        Plushie
+    }
+
+    [Dependency] private readonly TagSystem _tag = default!;
```

Comment:
to the top of the system

Reply:
I've seen precedence for doing it this way, global variables above dependencies. I can switch it.